### DISCUSSION TITLE
brew bump --open-pr option to manually provide version

### DISCUSSION THREAD

**Discussion by xyzzy-plugh-plover** [Homebrew/discussions#3878] [Everyday usage]
Sometimes a cask or formula doesn't have a working livecheck, but you know the new version info.

It would be useful to be able to manually provide the version to brew bump --open-pr via an option (possibly named `--version`), like:

```
brew bump --open-pr --version <version> <cask/formula name>
```

**Comment by carlocab** [comment]
Have you tried using
```
brew bump-formula-pr --version=1.2.3 foo
```
or
```
brew bump-cask-pr --version=1.2.3 bar
```
?

**Reply by p-linnane** [reply]
To add to this, if the `livecheck` isn't working that will also need to be fixed before it's merged.

**Reply by xyzzy-plugh-plover** [reply]
Thanks, that works. It would, however, be nice to add the `--version=<version>` option to `brew bump --open-pr` so that there would be a single simplified CLI.

**Reply by SMillerDev** [reply]
But `brew bump` isn't meant for that in the same way that it would be nice if `git merge` also committed code but doesn't. `brew bump` updates a package based on the livecheck result, no `version` required.
